AutomatedTesting/Gem/PythonTests/Terrain/TestSuite_Main.py

```diff
@@ -0,0 +1,25 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+"""
+
+# This suite consists of all test cases that are passing and have been verified.
+
+import pytest
+import os
+import sys
+
+from ly_test_tools import LAUNCHERS
+from ly_test_tools.o3de.editor_test import EditorTestSuite, EditorSingleTest
+
+@pytest.mark.SUITE_main
+@pytest.mark.parametrize("launcher_platform", ['windows_editor'])
+@pytest.mark.parametrize("project", ["AutomatedTesting"])
+class TestAutomation(EditorTestSuite):
+    global_extra_cmdline_args=["--regset=/Amazon/Preferences/EnablePrefabSystem=true"]
+
+    class test_AxisAlignedBoxShape_ConfigurationWorks(EditorSingleTest):
```

Comment:
Nit: This *should* be able to be run as an EditorSharedTest, although with it being the only test in the suite as of now, that can be updated later and this is functionally the same thing for now.

Reply:
I'll change this ready.